openmetadata-ui/src/main/resources/ui/src/pages/UserListPage/UserListPageV1.tsx

```diff
@@ -182,7 +191,20 @@ const UserListPageV1 = () => {
   useEffect(() => {
     initialSetup();
     if (teamsAndUsers.includes(tab)) {
-      fetchUsersList(tab === GlobalSettingOptions.ADMINS || undefined);
+      // Checking if the path has search query present in it
+      // if present fetch userlist with the query
+      // else get list of all users
+      if (location.search) {
+        // Converting string to URLSearchParameter
+        const searchParameter = new URLSearchParams(location.search);
+        // Getting the searched name
+        const searchTerm = searchParameter.get('name') || '';
```

Comment:
`name` isn't correct searchTerm that we can have in the URL
Try using `search` instead

Reply:
Done 👍